src/sentry/static/sentry/app/components/events/eventMessage.tsx

```diff
@@ -21,14 +23,24 @@ const EventMessage = ({
   levelIndicatorSize,
   message,
   annotations,
+  hasGuideAnchor,
 }: Props) => (
   <div className={className}>
     {level && (
       <StyledErrorLevel size={levelIndicatorSize} level={level}>
         {level}
       </StyledErrorLevel>
     )}
-    {message && <Message>{message}</Message>}
+
+    {message &&
+      (hasGuideAnchor ? (
+        <GuideAnchor target="issue_title" position="bottom">
```

Comment:
Would be nice to have a `disabled` (or `enabled`) prop on `<GuideAnchor>` so that it can just render the `children` if disabled. Would remove some duplication when using it like this.